src/middlewares/checkIfCommentExist.js

```diff
@@ -0,0 +1,39 @@
+import models from '../api/models';
+import statusCode from '../helpers/constants/status.codes';
+import sendError from '../helpers/error.sender';
+
+const { Comment } = models;
+/**
+ * check Comment middleware
+ *
+ * @export
+ * @class checkIfCommentExist
+ */
+class checkIfCommentExist {
+  /**
+   * this is a middleware which checks if comment exist.
+   *
+   * @author R
+   * @static
+   * @param {object} req the request
+   * @param {object} res the response to be sent
+   * @param { object } next the next route controller to be called
+   * @memberof checkIfCommentExist
```

Comment:
I have a middleware doing almost the same thing. I think there is a way we can merge them into one

Reply:
@truestbyheart has a function to feed all middlewares,
I think after @MemunaHaruna merge this PR, this middleware will get refactored.